### PR TITLE
Add proper Windows location for pg_service.conf 

### LINKED ISSUE #26: Setup on windows
I tried to follow the docs on windows, creating a file at `%HOMEPATH%\.pg_service.conf` didn't work for me (it was not consumed by QGIS).
I had to set the `PGSERVICEFILE` env variable instead to make it work.

### PR DESCRIPTION
**Pull request by gacarrillor**
Namely, at `%APPDATA%/postgresql/.pg_service.conf`, and avoid the `~/.pg_service.conf` location for Windows, since clients like QGIS won't go there to find the file.

Fix #26

### CODE REVIEW

File: pg_service_parser/core/pg_service_parser_wrapper.py [hunk 1]
```diff
@@ -1,50 +1,62 @@
...
-    return path if path.exists() else None
+    pg_config_path = None
+    if getenv("PGSERVICEFILE"):
+        pg_config_path = Path(getenv("PGSERVICEFILE"))
+    elif getenv("PGSYSCONFDIR"):
+        pg_config_path = Path(getenv("PGSYSCONFDIR")) / "pg_service.conf"
+    else:
+        if platform.system() == "Windows":
+            pg_config_path = Path(getenv("APPDATA")) / ".pg_service.conf"
+        else:  # Linux or Darwin (TODO: verify Mac)
```
Comment (3nids) [inline pg_service_parser/core/pg_service_parser_wrapper.py:18]: This is fine on mac

**Comment by 3nids** [comment]
I'd suggest to have this in the lib directly indeed

File: pg_service_parser/core/pg_service_parser_wrapper.py [hunk 2]
```diff
@@ -1,50 +1,62 @@
...
-    service_name: str, settings: dict, conf_file_path: Optional[str] = None
+    service_name: str, settings: dict, conf_file_path: Optional[Path] = None
 ) -> bool:
     config = pgserviceparser.full_config(conf_file_path)
     if service_name in config:
         return False
 
     config.add_section(service_name)
-    with open(conf_file_path or pgserviceparser.conf_path(), "w") as f:
+    with open(str(conf_file_path), "w") as f:
```
Comment (3nids) [inline pg_service_parser/core/pg_service_parser_wrapper.py:48]: why do you need `str()` here?

File: pg_service_parser/core/pg_service_parser_wrapper.py (hunk 1 above)
Comment (gacarrillor) [reply pg_service_parser/core/pg_service_parser_wrapper.py:18]: Thanks!

**Comment by gacarrillor** [comment]
> I'd suggest to have this in the lib directly indeed

See PR for lib: https://github.com/opengisch/pgserviceparser/pull/26

**Comment by gacarrillor** [comment]
Rebasing to solve conflicts with other recently merged PRs.

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
Thanks @gacarrillor

**Pull request merged**